### PR TITLE
454: Dropdown Component

### LINKED ISSUE #454: Create dropdown config.

### LINKED ISSUE #455: Create dropdown template.

### LINKED ISSUE #456: Create dropdown component.

### PR DESCRIPTION
**Pull request by tamfleet**
Closes #454 
Closes #455 
Closes #456

### CODE REVIEW

File: src/components/dropdown.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export class BossyDropdown {
+	@Input() config: BossyDropdownConfig;
+	type: string = 'Button';
+	isSplit: boolean = false;
+	isDropup: boolean = false;
+	isRightAligned: boolean = false;
+	variant: string = 'secondary';
+	size: string = undefined;
+
+	ddShow(event) {
```
Comment (tabersays) [inline src/components/dropdown.ts]: I do not know what dd stands for, perhaps drop down? Please be more explicit.

File: src/config/dropdown-menu.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export class BossyDropdownMenuItemConfig {
+	constructor(
+		public type: string, // Use class instead of type for name?
+		public name?: string,
+		public href?: string,
+		public disabled?: boolean,
```
Comment (tabersays) [inline src/config/dropdown-menu.ts]: Booleans should be questions, isDisabled

File: src/config/dropdown-menu.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export class BossyDropdownMenuItemConfig {
+	constructor(
+		public type: string, // Use class instead of type for name?
```
Comment (tabersays) [inline src/config/dropdown-menu.ts]: I am unsure what you are asking.

Comment (lcramer3) [inline src/config/dropdown-menu.ts]: Old comment - removed it.

**Comment by glawson** [comment]
closes #454.

**Pull request merged**